src/console/controllers/DbController.php

```diff
@@ -82,6 +82,45 @@ public function options($actionID): array
         return $options;
     }
 
+    /**
+     * Repairs all tables in the database.
+     *
+     * Note that this can cause table locking, which could interfere SQL being executed.
+     *
+     *  Example:
+     *  ```
+     *  php craft db/repair
+     *  ```
+     *
+     * @since 4.15
+     * @see https://dev.mysql.com/doc/refman/8.4/en/optimize-table.html
+     * @see https://www.postgresql.org/docs/current/sql-analyze.html
+     */
+    public function actionRepair(): int
+    {
+        if (!$this->_tablesExist()) {
+            $this->stdout('No existing database tables found.' . PHP_EOL, Console::FG_YELLOW);
+            return ExitCode::OK;
+        }
+
+        if ($this->interactive && !$this->confirm('Are you sure you want to repair all tables from the database?')) {
```

Comment:
Is it possible to reach to the database name here (without querying the actual db, just config-wise) and embed it into the message?

Reply:
Possible - but we don't do it in any other console commands mainly because 99.9% of all Craft installs connect to a primary database and there's no confusion.